tests/test_specs.py

```diff
@@ -61,28 +51,26 @@ def test_two_point_line() -> None:
 
 def test_two_point_stepped_line() -> None:
     inst = step(Line(x, 0, 1, 2), 0.1)
-    dimx, dimt = inst.calculate()
-    assert dimx.midpoints == dimx.lower == dimx.upper == {x: approx([0, 1])}
-    assert dimt.midpoints == dimt.lower == dimt.upper == {DURATION: approx([0.1])}
-    assert inst.frames().gap == ints("11")
+    (dim, dimt) = inst.calculate()
+    assert dim.midpoints == {x: approx([0, 1])}
+    assert dim.gap == ints("10")
+    assert dimt.duration == approx([0.1, 0.1])
```

Comment:
This should be a single point

```suggestion
    assert dimt.duration == approx([0.1])
```